### DISCUSSION TITLE
Getting the pushname of contact

### DISCUSSION THREAD

**Discussion by rbcetin** [WhiskeySockets/Baileys#101] [New WhatsApp Features]
`const ppUrl = await sock.profilePictureUrl("xyz@g.us", 'image')`

Something like this but for the pushName. So for example when I parse the quotedMessage from a message, i don't know the pushName of the message owner. On _whatsapp-web.js_ I can easily retrieve the name of the contact.

**Comment by lomeliDev** [comment]
Were you able to do it?
